llvm_ir_dataset_utils/compile_time_analysis_tools/datatset_download/write_data_files.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: ascii -*-
+from datasets import load_dataset, parallel
+import os
+import multiprocessing
+import csv
+from sys import argv
+
+# Usage:
+# python write_data_files.py [STORAGE]
+
+STORAGE: str
+if len(argv) > 1:
+    STORAGE = argv[1]
+else:
+    STORAGE = '/tmp'
```

Comment:
We should explicitly error out if a storage location is not set rather than defaulting it to `/tmp`.